packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,13 @@
 # @labkey/components
 
 Components, models, actions, and utility functions for LabKey applications and pages.
+
+### version 0.??.0
+*Released*: ?? December 2019
+* Add selectionKey getter to QueryGridModel
+* Add `showSampleComparisonReports`, `onChartClicked`, and `onPreviewSCRClicked` props to QueryGridPanel.
```

Comment:
@labkey-alan This release note should be updated after your change to these props.

Reply:
Good catch!